devicon.json

```
@@ -7444,6 +7444,27 @@
         "color": "#000000",
         "aliases": []
     },
+    {
+        "name": "processwire",
+        "tags": [
+            "cms",
+            "php",
+            "framework"
+        ],
+        "versions":
+        {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain-wordmark"
+            ],
+            "font": [
+                "plain-wordmark"
+            ]
+        },
+        "color": "#e81d60",
+        "aliases": []
```

Add alias from `original` to `plain`
```suggestion
                "original",
                "plain-wordmark"
            ]
        },
        "color": "#e81d60",
        "aliases": [
            {
                "base": "original,
                "alias": "plain"
            }
        ]
```